### ISSUE TITLE
Should resolve ~ in path to home directory

### ISSUE THREAD

**Issue by garysweaver** [gstroup/apimocker#23]
If mockDirectory in config.json starts with '~', e.g. `"mockDirectory": "~/some/path/"`, it should resolve the tilde to the current user's home directory.

E.g. if config.json looks like:

```
{
  "mockDirectory": "~/some/path/",
  "quiet": false,
  "port": "1234",
  "latency": 50,
  "allowedDomains": ["abc.com"],
  "allowedHeaders": ["Content-Type", "my-custom-header"],
  "webServices": {
    "a/b/c": {
      "mockFile": "some_file.json",
      "latency": 20,
      "verbs": ["get"]
    }
  }
}
```

And in `~/some/path/some_file.json` you have:
```
{
  "this": "worked"
}
```

If you start apimocker and try to GET that provided service, it has an error, e.g.: `Error: ENOENT, stat &#39;~/some/path/some_file.json&#39;`.

A workaround is to modify the config prior to use to replace ~ in the mockDirectory value of config.json with the home directory path, e.g.:

```
mkdir -p tmp/apimock
sed 's#~#'$HOME'#g' config.json > tmp/apimock/config.json
apimocker -c tmp/apimock/config.json
```

**Comment by gstroup** [comment]
Hmm... the tilde ~ works for me on Mac and Linux.  Are you working on Windows?  I think I found a fix for this... Let me get on a Windows box and verify.

**Comment by garysweaver** [comment]
@gstroup having this problem in OS X with 0.4.1:

```
$ apimocker --version
0.4.1
```

**Comment by gstroup** [comment]
Gary, I didn't read your initial issue correctly.  I found the problem you mentioned and fixed it in v 0.4.2.  Let me know if you see any problems.  Thanks.  -Greg

**Comment by garysweaver** [comment]
Thanks, @gstroup!
